src/types/actions.js

```diff
@@ -89,7 +89,11 @@ type ProfileAction =
       selectedMarker: IndexIntoMarkersTable | -1,
     }
   | { type: 'UPDATE_PROFILE_SELECTION', selection: ProfileSelection }
-  | { type: 'CHANGE_TAB_ORDER', tabOrder: number[] };
+  | { type: 'CHANGE_TAB_ORDER', tabOrder: number[] }
+  | {
```

Comment:
We still need to update all the old code, but I'd prefer all new actions to be exact and read only. We still need to convert all the existing ones, but that would be a different PR.

e.g.

```js
  | {|
      +type: 'SET_PROFILE_CALL_TREE_CONTEXT_MENU_VISIBILITY',
      +isVisible: boolean,
    |};
```